src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
+    // As this point, it MUST have a name, as we dealt with the special ones already
+    if (!actuatedComponent_->name()) {
+      LOG(Error, "Actuated Component Name for Actuator '" << modelObject.nameString() << "' does not exist, it will not be translated.");
+      return boost::none;
+    }
+
+    // check if actuatedComponent is a SpaceLoad
+    auto load_ = actuatedComponent_->optionalCast<model::SpaceLoadInstance>();
+    if (!load_) {
+      // Classic case, we just write it
+      IdfObject idfObject = createRegisterAndNameIdfObject(openstudio::IddObjectType::EnergyManagementSystem_Actuator, modelObject);
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, actuatedComponent_->nameString());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
+      return idfObject;
+    }
+
+    // SpaceLoadInstance: check if first if it's assigned to a single space
+    auto space_ = load_->space();
+    if (space_) {
+      // if load is assigned to a single space instead of a space type then the name will not be changed in EnergyPlus
+      IdfObject idfObject = createRegisterAndNameIdfObject(openstudio::IddObjectType::EnergyManagementSystem_Actuator, modelObject);
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, actuatedComponent_->nameString());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
+      return idfObject;
+    }
+
+    // At this point the SpaceLoadInstance is not assigned to a space, so it's assigned to a SpaceType or nothing
+
+    auto spaceType_ = load_->spaceType();
+    if (!spaceType_) {
+      LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                              << "' which is not associated with a ThermalZone, it will not be translated.");
+      return boost::none;
+    }
+
+    // Deal with the SpaceType case...
+
+    // Case 1: the user correctly assigned a Space or ThermalZone to it
+    if (const boost::optional<Space> specificSpace_ = modelObject.space()) {
+      // Assert the space has a thermalZone attached to it.
+      auto tz_ = specificSpace_->thermalZone();
+      if (!tz_) {
+        LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                << "' and specified that it wanted to be attached to the Space '" << specificSpace_->nameString()
+                                << "' but the Space has no Thermal Zone, so it will not be translated.");
```

Comment:
Alright, the user assigned a "Space" (not a Zone). If that Space has no zone, we have no business translating it.